prototypes/artillery-cluster-remote.lua

```diff
@@ -150,12 +149,33 @@ for ammo_category, tint_colour in pairs(ammo_categories) do
     order = "b[turret]-d[artillery-turret]-ba[remote]",
     flags = { "only-in-cursor", "not-stackable", "spawnable" },
     stack_size = 1,
+    inventory_move_sound = {
+      filename = "__base__/sound/item/artillery-remote-inventory-move.ogg",
+      volume = 0.7,
+      aggregation = {max_count = 1, remove = true},
+    },
+    pick_sound = {
+      filename = "__base__/sound/item/mechanical-inventory-pickup.ogg",
+      volume = 0.8,
+      aggregation = {max_count = 1, remove = true},
+    },
+    drop_sound = {
+      filename = "__base__/sound/item/artillery-remote-inventory-move.ogg",
+      volume = 0.7,
+      aggregation = {max_count = 1, remove = true},
+    },
   }
 
+  local associated_control_input = nil
+  if ammo_category == "artillery-shell" then
```

Comment:
This would not support having control inputs for different cluster remotes, but I can revisit this part after PR merge as well (I guess there might be some mess around localisation).